tests/unit/test_readers.py

```diff
@@ -34,45 +18,25 @@ def test_read_invalid_extenstion() -> None:
 def test_gt3x_loader() -> None:
     """Test the gt3x loader."""
     test_file = pathlib.Path(__file__).parent / "sample_data" / "example_actigraph.gt3x"
-    watch_data = readers.gt3x_loader(test_file)
+    watch_data = readers.read_watch_data(test_file)
     assert isinstance(watch_data, WatchData)
     assert isinstance(watch_data.acceleration, Measurement)
     assert isinstance(watch_data.lux, Measurement)
```

Comment:
I think we've been importing the models module instead of the individual functions/classes so I'd rewrite the imports and call models.WatchData and models.Measurement

Reply:
True, this was written before that and never reflected that change. Will update.